src/components/VentSwitch/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Colors from '../../constants/Colors';
+
+import {
+  ConfigLabelText,
+  ConfigLableContainer,
+  ConfigMetricWrapper,
+  SwitchContainer,
+  Switch,
+} from '../Globals/ConfigMetric';
+
+interface VentSwitchProps {
+  switchHandler: (value : boolean) => void;
+  isVentilating: boolean;
+}
+
+const VentSwitch = ({ switchHandler, isVentilating }: VentSwitchProps) => {
+  return (
+    <ConfigMetricWrapper style={{
+      marginLeft: 'auto',
+      marginRight: 35,
+      justifyContent: 'space-between',
+      width: '35%'
+    }} borderBottom='transparent' >
+      <ConfigLableContainer style={{
+        borderRightWidth: 0,
+        flex: 1 
+      }}>
+        <ConfigLabelText size='25px'>
```

Comment:
You should say `Ventilation Started` in case of `isVentilating` and `Ventilation Stopped` in case of the opposite